tso_session/ExploringTSOProducts.ipynb

```diff
@@ -114,15 +114,87 @@
     "\n",
```

Comment:
<p>You can ignore the first two warnings here; <code>flake8</code>  isn't recognizing that the <code>%set_env</code>  statements are not Python.</p><p> </p><p>You can get rid of these warnings by either separating environment variable assignment into its own cell or by moving it after the imports in this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>